device_doctor/lib/src/ios_device.dart

```diff
@@ -66,7 +68,7 @@ class IosDeviceDiscovery implements DeviceDiscovery {
     }
   }
 
-  Future<HealthCheckResult> _keychainUnlockCheck({ProcessManager processManager}) async {
+  Future<HealthCheckResult> keychainUnlockCheck({ProcessManager processManager}) async {
```

Comment:
Should these checks have `@visibleForTesting` decoration?

Reply:
Good point. Added.